src/main/drivers/dshot_bitbang_ll.c

```diff
@@ -142,6 +143,25 @@ static void bbSaveDMARegs(dmaResource_t *dmaResource, dmaRegCache_t *dmaRegCache
     dmaRegCache->PAR = ((DMA_ARCH_TYPE *)dmaResource)->PAR;
     dmaRegCache->M0AR = ((DMA_ARCH_TYPE *)dmaResource)->M0AR;
 }
+#elif defined(STM32G4)
+void bbLoadDMARegs(dmaResource_t *dmaResource, dmaRegCache_t *dmaRegCache)
```

Comment:
The function signature can be pulled out of the conditional for better maintainability:

```
void bbLoadDMARegs(dmaResource_t *dmaResource, dmaRegCache_t *dmaRegCache)
￼{
#if defined(STM32F7)
[...]
#elif defined(STM32G4)
[...]
```